packages/backend/src/controllers/authentication.ts

```diff
@@ -187,18 +200,22 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
                           issuerType: OpenIdIdentityIssuerType.GOOGLE,
                       },
                   };
+
                   const user = await userService.loginWithOpenId(
                       openIdUser,
                       req.user,
                       inviteCode,
+                      refreshToken,
```

Comment:
save refreshtoken on the db

Reply:
We could still store the scope it should just be `openid` or maybe `openid profile`